### ISSUE TITLE
App always add yourself into autostart at launch on linux

### ISSUE THREAD

**Issue by niko-lay** [trustcrypto/OnlyKey-App#146]
App (5.3.1) always creates file `~/.config/autostart/nw.desktop` on launch, even menu option,  "Auto-launch app on system login" is disabled.
```
$ cat /etc/*release*
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

$ echo $XDG_CURRENT_DESKTOP
KDE
```
is this expected behavior?

**Comment by niko-lay** [comment]
here is [issue video ](https://user-images.githubusercontent.com/5228230/105517626-f6064300-5cdf-11eb-8623-ee0e323777c3.mp4)

**Comment by niko-lay** [comment]
fixed in #184
